tutorials/snowplow-cli-mcp/introduction.md

```diff
@@ -3,7 +3,7 @@ position: 1
 title: Introduction
 ---
 
-# Getting started with Snowplow CLI MCP tool
+# Getting Started with the Snowplow MCP Server for Tracking Design
```

Comment:
```suggestion
```

Reply:
let's rename this tutorial to "Tracking design with the Snowplow MCP server" or similar instead

Reply:
Ahh, you mean like the other ones you said the same thing about and I fixed and have forgotten about in the last 24 hours. Sorry, I failed. How about now?

Reply:
that was hours ago! yes great